apps/frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,44 @@
+import { useForm } from "@mantine/form";
+import { TextInput, Button, Container, Anchor, Title, Space } from "@mantine/core";
+import { useRegister } from "../features/auth/lib/auth";
+
+interface FormValues {
+	email: string;
+}
+
+export const Login = (): React.ReactElement => {
+	const form = useForm<FormValues>({
+		mode: "uncontrolled",
+		initialValues: { email: ""},
+	});
+
+	const registering = useRegister();
+
+	return (
+        <div>
+           <Title ta="center">Login</Title>
```

Comment:
All visible text should be pulled from `packages/lib/i18n/translations.ts` 
See `useTranslation` in `apps/frontend/src/features/schedule/components/Schedule.tsx` how to use it